### PR TITLE
feat, python: support envvar scanning for headers

### PR DESCRIPTION
**Pull request by armandobelardo**

### CODE REVIEW

File: generators/python/sdk/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,37 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.11.6] - 2024-02-26
+
+- Improvement: You can now specify envvars to scan for headers, not just auth scheme headers.
+  ```
+  x-fern-global-headers:
```
Comment (dsinghvi) [inline generators/python/sdk/CHANGELOG.md:12]: nit: lets label with `# OpenAPI`

File: generators/python/sdk/CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,37 @@ All notable changes to this project will be documented in this file.
...
+   - header: x-api-key
+     name: api_key
+     optional: true
+     env: MY_API_KEY
+  ```
+  ... or ...
+  ```
```
Comment (dsinghvi) [inline generators/python/sdk/CHANGELOG.md:19]: nit: lets label with `# Fern Definition`

File: seed/python-sdk/auth-environment-variables/src/seed/client.py [hunk 3]
```diff
@@ -26,6 +28,7 @@ class SeedAuthEnvironmentVariables:
     from seed.client import SeedAuthEnvironmentVariables
 
     client = SeedAuthEnvironmentVariables(
+        x_another_header="YOUR_X_ANOTHER_HEADER",
```
Comment (dsinghvi) [inline seed/python-sdk/auth-environment-variables/src/seed/client.py:31]: @armandobelardo (for the future) we should do something smart in the IR Generation that removes `x_` for any headers

**Review by dsinghvi — APPROVED** [review]
🚀

File: seed/python-sdk/auth-environment-variables/src/seed/client.py (hunk 3 above)
Comment (dsinghvi) [reply seed/python-sdk/auth-environment-variables/src/seed/client.py:31]: (for the header name)

Comment (armandobelardo) [reply seed/python-sdk/auth-environment-variables/src/seed/client.py:31]: Ah that's fair, I thought we did it now, but I think it's only for openAPI headers, not ones from Fern defs

**Pull request merged**